Alacritty fails to start

# Alacritty configuration error: unknown keyboard action 'SelectAll'

## Issue Description

The Alacritty configuration file `windows-keybindings.toml` contains an invalid keyboard action that causes Alacritty to fail to start properly.

## Error Details

When running Alacritty with the current configuration, the following error occurs:

```
[ERROR] [alacritty_config_derive] Config error: unknown keyboard action 'SelectAll'
; ignoring binding
```

## Affected File

`configs/alacritty/windows-keybindings.toml`

## Problematic Code

```toml
[[keyboard.bindings]]
key   = "A"
mods  = "Control"
action = "SelectAll"  # This action is not recognized
```

## Environment

- Alacritty version: 0.13.2
- Ubuntu Linux
- Omakub/Ardbegian setup

## Expected Behavior

The `Ctrl+A` keybinding should select all text in the terminal.

## Possible Solutions

1. Replace `SelectAll` with the correct action name for Alacritty 0.13.2
2. Check Alacritty documentation for the proper action name
3. Consider using a different approach for select-all functionality

## Additional Context

This issue was discovered when trying to start Alacritty on a fresh Omakub installation. The configuration import path was also incorrect (pointing to local config instead of Omakub config directory), but that was a separate issue.

## Steps to Reproduce

1. Install Omakub
2. Try to start Alacritty
3. Check logs for configuration errors

--- a/configs/zellij.kdl
+++ b/configs/zellij.kdl
@@ -1,6 +1,6 @@
-theme "tokyo-night"
-default_layout "compact"
-on_force_close "quit"
+theme "tokyo-night";
+default_layout "compact";
+on_force_close "quit";
 
 # Status bar configuration
 status_bar {
@@ -20,7 +20,7 @@ status_bar {
     }
 }
 
-default_mode "locked"
+default_mode "locked";
 keybinds clear-defaults=true {
     locked {
         bind "Ctrl g" { SwitchToMode "normal"; }

